ChatLayout/Classes/Core/ChatLayoutAttributes.swift

```diff
@@ -16,13 +16,13 @@ import UIKit
 /// Custom implementation of `UICollectionViewLayoutAttributes`
 public final class ChatLayoutAttributes: UICollectionViewLayoutAttributes {
     /// Alignment of the current item. Can be changed within `UICollectionViewCell.preferredLayoutAttributesFitting(...)`
-    public var alignment: ChatItemAlignment = .fullWidth
+    public nonisolated(unsafe) var alignment: ChatItemAlignment = .fullWidth
```

Comment:
It is not possible to set `ChatLayoutAttributes` to be `@MainActor`? That would feel intuitively safer to me.

Reply:
@Frizlab Hey. Thank you. It is possible and it is actually inherited from `UICollectionViewLayoutAttributes` but NSObjects `isEqual` is not guaranteed that it will be called on main thread, even logically `UICollectionView` wont call it not on main thread. But I did not find another way to express it.